src/jormungandr/testing/jormungandr-integration-tests/src/jormungandr/explorer/certificates.rs

```diff
@@ -729,6 +729,7 @@ pub fn explorer_vote_tally_certificate_test() {
 
     wait_for_epoch(1, jormungandr.rest());
 
+    fragment_builder.update_valid_until(BlockDate::first().next_epoch().next_epoch());
     let vote_tally_fragment =
```

Comment:
Are you sure is this needed? BlockDate::first().next_epoch() is the 2nd epoch and we wait until the first epoch

Reply:
Yes, you are correct and as I understood correctly after `wait_for_epoch(1, jormungandr.rest());` we are in the 2 epoch  .
At least here is the node log `incoming_fragment{fragment_id=Hash(Blake2b256(0x8fd27387bc3a7635fb63f41170439afaf11667a17f6064d36c5e8df905e26e6a))}:  jormungandr::fragment::pool: fragment is expired at the time of receiving `